test/integration/txes/create_account_test.cpp

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include "backend/protobuf/transaction.hpp"
+#include "block.pb.h"
+#include "builders/protobuf/transaction.hpp"
+#include "cryptography/crypto_provider/crypto_defaults.hpp"
+#include "datetime/time.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "interfaces/utils/specified_visitor.hpp"
+#include "module/shared_model/validators/validators.hpp"
+
+using namespace std::string_literals;
+using namespace integration_framework;
+using namespace shared_model;
+
+class CreateAccount : public ::testing::Test {
+ public:
+  const std::string kUser = "user"s;
+  const crypto::Keypair kAdminKeypair =
+      crypto::DefaultCryptoAlgorithmType::generateKeypair();
+  const crypto::Keypair kUserKeypair =
+      crypto::DefaultCryptoAlgorithmType::generateKeypair();
+};
+
+/**
+ * @given tx with CreateAccount command and invalid signature size
+ * @when send it
+ * @then Torii returns stateless fail
+ */
+TEST_F(CreateAccount, InvalidSignature) {
+  auto tx = proto::TransactionBuilder()
+                .createAccount(kUser, "test", kUserKeypair.publicKey())
+                .txCounter(1)
+                .creatorAccountId("admin@test")
+                .createdTime(iroha::time::now())
+                .build()
+                .signAndAddSignature(kAdminKeypair)
+                .getTransport();
+  tx.mutable_signature(0)->mutable_signature()->operator+=("asd");
```

Comment:
Looks bit unclear

Reply:
Should be `mutable_signatureS`, typo in block.proto
This line makes first signature invalid, I'll add a comment